src/utils/onLink.js

```diff
@@ -72,9 +72,14 @@ const onLink = (config, handler) => {
     // 2. rel="external" attribute
     if (element.hasAttribute('download') || element.getAttribute('rel') === 'external') { return false; }
 
+    const originalHref = element.getAttribute('href');
+    const sanitizedHref = originalHref.toLowerCase();
+
     // Check for mailto: in the href
-    const href = element.getAttribute('href');
-    if (href && href.indexOf('mailto:') > -1) { return false; }
```

Comment:
`href` is always a `string` no need to check for its presence.

Reply:
Wouldn't `href` be null if `getAttribute('href')` is called on an element without an `href` attribute?

Reply:
**tl;dr** `LinkElement#href` will always be `''`, `getAttribute('href')` will effectively be `null`

So that's true if like here we're using `getAttribute` which I should have also removed.

We check that it's a link element above, so at this point we know it's a link. Link element always come with a`href` property set to `''` by default. However you're right `Element#getAttribute` will seek for an attribute provided by the markup.

Switching to `element.href` will be safe, nice catch!